NSS-Day-Cohort-71/modern-farm-tiredteam
# Created seeds modules and functions in the scripts/seeds directory

## What?
I created a 'seeds' directory inside of the 'src/scripts' directory. Then I created 6 modules inside the src/scripts/seeds directory to represent each kind of crop type, as well as functions inside each module to generate objects or arrays with relevant data about each crop. 

## Why?
These modules generate data that can be stored for use later in our project.

## How?
Inside each of the modules except 'corn.js', I created and exported a function that defines an object with three properties--'type', 'height', and 'output'-- and assigns values to those properties depending on the crop. In the 'corn.js' module, I created and exported a function that defines and returns an array with 2 identical objects with 'type', 'height', and 'output' properties. 
1. asparagus.js - createAsparagus(); return asparagusObj 
2. wheat.js - createWheat(); return wheatObj
3. potato.js - createPotato(); return potatoObj
4. soybean.js - createSoybean(); return soybeanObj
5. sunflower.js - createSunflower(); return sunflowerObj
6. corn.js - createCorn(); return cornArr

I used this chart to define values for each crop:

**Type | Height | Output** 
Soybean | 20 | 4 
Corn | 180 | 6 
Sunflower | 380 | 3 
Asparagus | 24 | 4
Wheat | 230 | 6
Potato | 32 | 2

## Testing?
- Navigate to project directory in the terminal
- Run serve command 
- Open the url provided in browser (i.e. localhost:3000)
- Open up dev tools
- Navigate to the 'Sources' tab, then the 'scripts' directory
- Open the 'main.js' file
- Run debugger & check console.log outputs in the browser console
*Check to see if the correct datatype (object or array) detailed in project instructions with the correct properties displays in the console
*Check that the values of the properties against the 'Plant Properties Table' found in project instructions to make sure they are correct
